ddp-workspace/projects/ddp-basil/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { SessionMementoService } from 'ddp-sdk';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+
+  constructor(private session: SessionMementoService) { }
+
+  ngOnInit(): void {
```

Comment:
should we remove it?

Reply:
Just leaving it here for now. My PR is incomplete, but hopefully next PRs will flush things out.